sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -861,7 +861,7 @@
           <name>generate-overview</name>
         </property>
         <file>
-          <exists>${project.basedir}/${relative.path.to.eng.folder}/eng/pipelines/scripts/generate_overview_from_readme.py</exists>
+          <exists>${basedir}/${relative.path.to.eng.folder}/eng/pipelines/scripts/generate_overview_from_readme.py</exists>
```

Comment:
found other places using {project.basedir}. is that matter?

Reply:
Depends on where `{project.basedir}` is being used. If it is for a profile activation as seen here it will need to be updated.